common-test-lib/src/main/java/com/google/gct/idea/testing/TestUtils.java

```diff
@@ -119,6 +120,16 @@ private static Disposable getParentDisposableForCleanup() {
   }
 
   @NotNull
+  public static <T> T installMockService(@NotNull Class<T> serviceInterface) {
```

Comment:
I'm seeing test failures for `CloudDebugProcessStateTest` and `CloudDebugProcessTest` due to a duplicated key error from the Pico Container:

Looks like the mock is being registered during @setUp. I suspect we need either unregister it first, or check for the component before registering it with the container.